Initial documentation for fyta integration (#31480)
home-assistant/home-assistant.github.io: source/_integrations/fyta.markdown

## Sensors

The following sensors are currently available for the controller:

| name                  | Unit   | Description   |
|-----------------------|--------|:-------------------------------------------|
| plant_number          |        | Number of plants                           |

The following sensors are currently available per plant:

| name                  | Unit   | Description   |
|-----------------------|--------|:-------------------------------------------|
| last_update           |        | Time of latest data update.                |
| plant_name            |        | Name of the plant (as set by user)         |
| scientific_name       |        | Scientific name of the plant               |
| plant_status          |        | FYTA-Status (number 1 to 5)                |
| temperature_status    |        | FYTA-Status (number 1 to 5)                |
| light_status          |        | FYTA-Status (number 1 to 5)                |
| moisture_status       |        | FYTA-Status (number 1 to 5)                |
| salinity_status       |        | FYTA-Status (number 1 to 5)                |
| temperature           | °C     | Temperature measured by sensor             |

> | light                 | mol/d  | Light measured by sensor                   |

Reviewer: is this referring to the day light integral DLI? 
![image](https://github.com/home-assistant/home-assistant.io/assets/38767475/812ab8a5-4923-449e-9dc5-5f45def913e1)

Author: No, the value provided by this API is the current value. It should also be possible to get the DLI, but this would be something for a new PR. While digging through the API and documentation, I noticed that I reported the wrong unit (mol/d instead of μmol/h), just amended the documentation accordingly.

Reviewer: I see. I was not trying to suggest to use DLI. I was simply trying to understand what I'm looking at. When I googled this, I always found it in connection with a certain area, for example per square meter (for example, in [this documentation on calculating light](https://edepot.wur.nl/156931)).

Author: I now also added an explanation of the unit

Reviewer: Nice! Thank you for clarifying.

| moisture              | %      | Moisture measured by sensor                |

> | salinity              | mS/cm  | Salinity measured by sensor                |

Reviewer: as measured by electrical conductivity?

Author: Yes, this is my understanding from: "Der Salzgehalt wird in elektrischer Leitfähigkeit gemessen und als Mikrosiemens pro Zentimeter (mS/cm) ausgedrückt."

Reviewer: Maybe it's worth adding a note? Maybe that's how salinity is usually measured. But when I saw the units, I was surprised and went to check if this is a copy-paste error.

---

> title: Fyta

Reviewer: ```suggestion
title: FYTA
```

> description: Instructions on how to integrate Fyta sensors within Home Assistant.

Reviewer: ```suggestion
description: Instructions on how to integrate FYTA sensors within Home Assistant.
```

ha_category:
  - Sensor
ha_release: 0.1
ha_iot_class: Cloud Polling
ha_config_flow: true
ha_codeowners:
  - '@dontinelli'
ha_domain: fyta
ha_platforms:
  - sensor
ha_integration_type: integration
---

> The fyta {% term integration %} uses the open API of [FYTA](https://www.fyta.de) to obtain the data from your plant sensors and integrate these into Home Assistant.

Reviewer: ```suggestion
The **FYTA** {% term integration %} uses the open API of [FYTA](https://www.fyta.de) to obtain the data from your plant sensors and integrate these into Home Assistant.
```
What is the integration name exactly? I see fyta, FYTA, and Fyta. Suggest to change it to FYTA, as it is spelled like this on their website. Please correct me if I got this wrong.

> The `fyta` integration uses the open APY of the [FYTA](https://www.fyta.de) to obtain the data from your plant sensors and integrate these into your Home Assistant installation.

Reviewer 2: ```suggestion
The fyta {% term integration %} uses the open API of [FYTA](https://www.fyta.de) to obtain the data from your plant sensors and integrate these into Home Assistant.
```

> For the authentication on the FYTA-server you need your login-credentials (email and password).

Reviewer: ```suggestion
For the authentication on the FYTA server, you need your login-credentials (email and password).
```

> The integration provides a device for the controller interface and all plants with a Fyta Beam sensor.

Reviewer: ```suggestion
The integration provides a device for the controller interface and for all plants with a FYTA Beam sensor.
```

> For the authentication on the FYTA-server you need your login-credentials (email and password). At a later stage the login via access token generated in the FYTA portal may also be supported.

Reviewer 2: ```suggestion
For the authentication on the FYTA-server you need your login-credentials (email and password).
```
We should describe the current state of the integration

| last_update           |        | Time of latest data update.                |
| plant_name            |        | Name of the plant (as set by user)         |
| scientific_name       |        | Scientific name of the plant               |
| plant_status          |        | FYTA-Status (number 1 to 5)                |
| temperature_status    |        | FYTA-Status (number 1 to 5)                |
| light_status          |        | FYTA-Status (number 1 to 5)                |
| moisture_status       |        | FYTA-Status (number 1 to 5)                |
| salinity_status       |        | FYTA-Status (number 1 to 5)                |
| temperature           | °C     | Temperature measured by sensor             |
| light                 | mol/d  | Light measured by sensor                   |
| moisture              | %      | Moisture measured by sensor                |
| salinity              | mS/cm  | Salinity measured by sensor                |
| battery_level         | %      | Battery level of the sensor                |

<div class='note'>
The fyta integration is using the fyta_cli package to get the data from the Fyta server (over the public API).

> </div>

Reviewer 2: This could be omitted

Author: Could or should?

Reviewer 2: Should 😉 This is information that can be interesting for devs, but the docs focus on the normal user and for them it is of no added value.
